webtty/webtty.go

```diff
@@ -184,6 +188,17 @@ func (wt *WebTTY) handleMasterReadEvent(data []byte) error {
 			return errors.Wrapf(err, "failed to decode received data")
 		}
 
+		// log.Printf("[wlog] %v %X\n", decodedBuffer[:n], decodedBuffer[:n])
+		utils.FormatWritesLog(decodedBuffer[:n], line)
+		if decodedBuffer[n-1] == 13 {
+			argumentsByte, err := json.Marshal(wt.arguments)
```

Comment:
Re-marshaling this each time seems wasteful. Can't we marshal it once and store the result?